src/pages/Farm/FarmOverview.tsx

```diff
@@ -61,13 +59,18 @@ FarmOverviewProps): JSX.Element {
               <TokenIcon symbol="WETH" alt="weth" />
             </>
           ) : (
-            farmTokens.map((token) => (
-              <TokenIcon
-                key={token.symbol}
-                symbol={token.symbol}
-                alt={token.symbol}
-              />
-            ))
+            poolTokens?.map((tokenAddress) => {
+              if (!chainId) return <div></div>
+              const token = getTokenByAddress(tokenAddress, chainId)
+              if (!token) return <div></div>
```

Comment:
If it makes sense, instead of returning <div></div> can you `.filter` with same condition before the `.map`?

Reply:
Because I think we don't able to use filter because the poolTokens is only address of token, not token.

Reply:
```
poolTokens?.filter((tokenAddress) => Boolean(getTokenByAddress(tokenAddress, chainId)).map((tokenAddress) =>  
{
  const token = getTokenByAddress(...
 ```
 
Something like this should work but you have to double-check if you want to do this. After filtering you will only have valid tokens so there will be no use for `if (!token)` case.